tryoutmanager/models.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3.6
+
+import os
+import uuid
+import sqlalchemy as sa
+
+from datetime import datetime
+from sqlalchemy import Column, String, Boolean, DateTime, Table, \
+                       ForeignKey, Enum, Index, UniqueConstraint
+from sqlalchemy.orm import relationship, sessionmaker
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.ext.declarative import declarative_base
+
+db = sa.create_engine(os.environ['DATABASE_URL'])
```

Comment:
I'll accept this for now, but this is going to be replaced with pulling it from the configuration file down the road. 

I'd rather avoid OS environment variables unless we have a standard way of deploying this, which we don't.